data/schema.yaml

```diff
@@ -122,17 +117,17 @@ components:
                   enum: [Headline, Text,altText, paragraph, Image, caption, Video, rawImage, rawVideo]
                 model:
                   oneOf:
-                  - $ref: '#/components/schemas/Article/properties/model/properties/blocks'
-                  - $ref: '#/components/schemas/Text'
-                  - $ref: '#/components/schemas/Image'
-                  - $ref: '#/components/schemas/Headline'
+                  - $ref: '#/components/schemas/article/properties/model/properties/blocks'
```

Comment:
Can `blocks` contain `blocks`? That doesn't seem right to me.

Reply:
Agreed this has been removed.